packages/mask/package.json

```diff
@@ -64,6 +66,7 @@
     "async-call-rpc": "^6.0.1",
     "bignumber.js": "^9.0.2",
     "bip39": "^3.0.2",
+    "boring-avatars": "^1.5.8",
```

Comment:
Remove this since it is installed in `packages/plugins/CyberConnect/package.json`